x-pack/plugins/security_solution/public/detections/pages/detection_engine/rules/translations.ts

```diff
@@ -1205,6 +1205,14 @@ export const INSTALL_RULE_BUTTON = i18n.translate(
   }
 );
 
+export const INSTALL_RULE_BUTTON_ARIA_LABEL = (ruleName: string) =>
+  i18n.translate('xpack.securitySolution.addRules.installRuleButton.ariaLabel', {
+    defaultMessage: 'Install "{ruleName}',
```

Comment:
Closing double quite is missing here

```suggestion
    defaultMessage: 'Install "{ruleName}"',
```

I'm not sure if consider double quotes localizable. If not I'd rather move quotes in values like

```ts
values: {
  ruleName: `"${ruleName}"`,
},
```

Reply:
Oh.. thank you for showing me my typo.